Assets/MixedRealityToolkit.Tests/PlayModeTests/PressableButtonTests.cs

```diff
@@ -327,6 +327,91 @@ public IEnumerator ScaleLocalDistances()
             yield return null;
         }
 
+        /// <summary>
+        /// This tests the release behavior of a button
+        /// </summary>
+        [UnityTest]
+        public IEnumerator ReleaseButton()
+        {
+            GameObject testButton = InstantiateSceneAndDefaultPressableButton();
+            TestUtilities.PlayspaceToOriginLookingForward();
+
+            PressableButton buttonComponent = testButton.GetComponent<PressableButton>();
+            Assert.IsNotNull(buttonComponent);
+
+            bool buttonPressed = false;
+            buttonComponent.ButtonPressed.AddListener(() =>
+            {
+                buttonPressed = true;
+            });
+
+            bool buttonReleased = false;
+            buttonComponent.ButtonReleased.AddListener(() =>
+            {
+                buttonReleased = true;
+            });
+
+            var inputSimulationService = PlayModeTestUtilities.GetInputSimulationService();
+            Vector3 startHand = new Vector3(0, 0, 0);
+            Vector3 inButtonOnPress = new Vector3(0, 0, 0.01f);
+            Vector3 rightOfButtonPress = new Vector3(1.0f, 0, 0.01f);
+            Vector3 inButtonOnRelease = new Vector3(0, 0, 0.005f);
+
+            // test scenarios in normal and low framerate
+            int[] stepVariations = { 30, 2 };
+            for (int i = 0; i < stepVariations.Length; ++i)
+            {
+                int numSteps = stepVariations[i];
+                // test release
+                yield return PlayModeTestUtilities.ShowHand(Handedness.Right, inputSimulationService);
```

Comment:
Suggestion: you can now use TestHand class Luis wrote to make hand movement easier.